### PR TITLE
add alert for pvcs

### PR DESCRIPTION
**Pull request by Talits**

### CODE REVIEW

**Comment by Talits** [comment]
[Alguns discos não aparecem no prometheus](https://trello.com/c/yAiMhUUc/137-alguns-discos-n%C3%A3o-aparecem-no-prometheus)

File: ansible/roles/getup-prometheus/templates/prometheus.getup.rules.j2 [hunk 1]
```diff
@@ -525,7 +525,19 @@ groups:
       description: 'Disk {{$label.device}} on {{$label.instance}} will be full in 7 days'
       summary: High disk usage
 {% endraw %}
-
+  - record: pvc:usage_percent
+    expr: kubelet_volume_stats_used_bytes{namespace=~"default|getup|kube-(public|system)|logging|management-infra|openshift|openshift-(grafana|infra|metrics|node|web-console)"}/kubelet_volume_stats_capacity_bytes{namespace=~"default|getup|kube-(public|system)|logging|management-infra|openshift|openshift-(grafana|infra|metrics|node|web-console)"} * 100
+  - alert: PVCisFULL
```
Comment (caruccio) [inline ansible/roles/getup-prometheus/templates/prometheus.getup.rules.j2]: Could you change it to `PVCUsage` in order to keep consistency with `NodeDiskUsage`?

**Comment by Talits** [comment]
done

File: ansible/roles/getup-prometheus/templates/prometheus.getup.rules.j2 [hunk 2]
```diff
@@ -525,7 +525,19 @@ groups:
...
+  - alert: PVCDiskUsage
+    expr: predict_linear(pvc:usage_percent[1h], 60 * 60 * 24 * 8) > 100
+    for: 24h
+    labels:
+      severity: critical
+      cluster: {{ cluster_name }}
+    annotations:
+{% raw %}
+    description: PVC {{$labels.persistentvolumeclaim}} in {{$labels.namespace}} will be full in 7 days - {{ cluster_name }}
+    summary: PVCisFULL
```
[the remark this answers is not in the document]
Comment (Talits) [reply ansible/roles/getup-prometheus/templates/prometheus.getup.rules.j2]: ok

**Pull request merged**